src/test/daml/ContingentClaims/Test/Pricing.daml

```diff
@@ -7,7 +7,7 @@ import ContingentClaims.Core.Builders (european)
 import ContingentClaims.Core.Claim (Claim, Inequality(..), anytime, one, scale)
 import ContingentClaims.Core.Observation qualified as O
 import ContingentClaims.Valuation.MathML qualified as MathML (ToXml, presentation)
-import ContingentClaims.Valuation.Stochastic (Expr(..), IsIdentifier(..), Process, fapf, gbm, riskless, {- simplify, -} unitIdentity)
+import ContingentClaims.Valuation.Stochastic (Expr(..), IsIdentifier(..), Process, fapf, gbm, riskless, unitIdentity)  -- simplify
```

Comment:
Is the comment required?

Reply:
Why do we need the two comments?

Reply:
Agree that it looks better without, so I removed them.